dist/leaflet/layers/leaflet-layer.directive.d.ts

```diff
@@ -11,11 +11,15 @@ import { LeafletDirective } from '../core/leaflet.directive';
 export declare class LeafletLayerDirective implements OnChanges, OnDestroy, OnInit {
     private zone;
     layer: Layer;
+    onAdd: EventEmitter<LeafletEvent>;
+    onRemove: EventEmitter<LeafletEvent>;
     private leafletDirective;
     constructor(leafletDirective: LeafletDirective, zone: NgZone);
     ngOnInit(): void;
     ngOnDestroy(): void;
     ngOnChanges(changes: {
         [key: string]: SimpleChange;
     }): void;
+    private handleEvent(eventEmitter, event);
```

Comment:
I see that down in the implementation, handleEvent has types for its arguments. Should the types be included here as well?

Reply:
I moved this into a static util class and made it a template function so it at least enforces that the type of the event and the type of the event emitter have to match.